src/validation/presenter.validation.ts

```diff
@@ -1,9 +1,9 @@
 import { body } from "express-validator";
 
 import { ErrorMessages } from "./error.messages";
-import { VALID_CHARACTERS } from "./regex/regex.validation";
+import { VALID_CHARACTERS, VALID_EMAIL_FORMAT } from "./regex/regex.validation";
 
 export const presenter = [
   body("full_name").not().isEmpty({ ignore_whitespace: true }).withMessage(ErrorMessages.FULL_NAME).isLength({ max: 160 }).withMessage(ErrorMessages.MAX_FULL_NAME_LENGTH).matches(VALID_CHARACTERS).withMessage(ErrorMessages.FULL_NAME_INVALID_CHARACTERS),
-  body("email").not().isEmpty({ ignore_whitespace: true }).withMessage(ErrorMessages.EMAIL).isLength({ max: 250 }).withMessage(ErrorMessages.MAX_EMAIL_LENGTH)
+  body("email").not().isEmpty({ ignore_whitespace: true }).withMessage(ErrorMessages.EMAIL).isLength({ max: 250 }).withMessage(ErrorMessages.MAX_EMAIL_LENGTH).matches(VALID_EMAIL_FORMAT).withMessage(ErrorMessages.EMAIL_INVALID_FORMAT)
```

Comment:
Hi, have you tried the [isEmail](https://express-validator.github.io/docs/custom-error-messages.html#complex-errors) check that comes with the `express-validator`.

Reply:
The checks absolutely everything including length so it depends on whether we want a separate error message for email max length or not.

Reply:
I would have changed only the 
```
.matches(VALID_EMAIL_FORMAT).withMessage(ErrorMessages.EMAIL_INVALID_FORMAT)
```
to something like this, if works 
```
.isEmail().withMessage(ErrorMessages.EMAIL_INVALID_FORMAT)
```

Reply:
Yep, that's is what I did. However, I have discussed this one message and we think this will be acceptable.